### PR TITLE
Migrate to PureScript 0.14

### PR DESCRIPTION
**Pull request by fsoikin**
This PR makes necessary changes to build with the shiny new PureScript 0.14.0 compiler.

### CODE REVIEW

**Comment by thomashoneyman** [comment]
Would you mind only including changes to the source code and to the bower file? You’ll want to (temporarily) point at your fork of spec in the bower file to avoid those conflicts you mentioned. Everything else should move up a major version.

Once the spec pr merges we can update that link and merge this pr, too.

File: package.json
Comment (thomashoneyman) [inline package.json:12]: You’ll want to bump Pulp to v15

File: package.json
Comment (thomashoneyman) [inline package.json:13]: I’d rather avoid adding spago in this PR

**Comment by fsoikin** [comment]
> Would you mind only including changes to the source code and to the bower file?

Do you mean no Spago and no changed in `package.json`?

**Comment by thomashoneyman** [comment]
That’s right, only changes to source code and to the bower.json file. If you’d like to migrate this library to use spago for local development I’d recommend doing it in a separate PR — or you could wait and see what @cdepillabout wants to do there — but I can merge this if you only do source code & bower changes.

**Comment by fsoikin** [comment]
Ok @thomashoneyman all done

**Comment by cdepillabout** [comment]
@fsoikin Thanks for sending this.

@thomashoneyman I'm completely happy to defer to your judgement for this.  It looks like it has been updated according to your recommendation, so I'm happy to merge it in once you sign off on it.

**Comment by fsoikin** [comment]
I just merged https://github.com/purescript-spec/purescript-spec/pull/113, published [version 5.0.0](https://pursuit.purescript.org/packages/purescript-spec/5.0.0), and updated `bower.json` in this PR.

File: bower.json
Comment (thomashoneyman) [inline bower.json:20]: Oh! I forgot to mention, we need to update this to be the same url as the registry for publishing to pursuit. In almost every case that just means changing this to https:// instead of git://

**Review by thomashoneyman — APPROVED** [review]
If you don’t mind updating that Bower url I’m happy to merge

**Comment by fsoikin** [comment]
Done

**Comment by thomashoneyman** [comment]
https://pursuit.purescript.org/packages/purescript-email-validate/6.0.0

**Comment by fsoikin** [comment]
Thank you very much @thomashoneyman!

**Pull request merged**
